### PR TITLE
Add an uber-department

### PR DESCRIPTION
**Pull request by pirj**
To avoid name and department clash issues, RuboCop decided to grant each extension its own department.

More info https://github.com/rubocop-hq/rubocop/pull/8490

---

Before submitting the PR make sure the following are checked:

* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [x] CHECK ONCE AGAIN, e.g. `Rails/PersistenceCalledOutsideExample`
* [-] Added tests.
* [-] Updated documentation.
* [-] Added an entry to the `CHANGELOG.md` if the new code introduces user-observable changes.
* [x] The build (`bundle exec rake`) passes (be sure to run this locally, since it may produce updated documentation that you will need to commit).

### CODE REVIEW

**Comment by bquorning** [comment]
@pirj I force-pushed to this branch, but the build is still failing. I’ll have a look at it later today.

**Comment by bquorning** [comment]
> the build is still failing. I’ll have a look at it later today.

I think I fixed it :-)

**Comment by pirj** [comment]
Thanks! LGTM.

**Comment by bquorning** [comment]
- Squashed into 2 commits.
- Added Changelog entries.
- Changed PR target branch.
- Specs are passing. 🎉 

I think we’re ready to merge.

File: config/default.yml [hunk 1]
```diff
@@ -620,33 +620,33 @@ RSpec/Yield:
   VersionAdded: '1.32'
   StyleGuide: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/Yield
 
-Capybara/CurrentPathExpectation:
+RSpec/Capybara/CurrentPathExpectation:
   Description: Checks that no expectations are set on Capybara's `current_path`.
   Enabled: true
   VersionAdded: '1.18'
```
Comment (bquorning) [inline config/default.yml:626]: Does this still apply when the cop is renamed?

Comment (pirj) [reply config/default.yml:626]: Let me quickly check how RuboCop itself does it.

Comment (pirj) [reply config/default.yml:626]: It seems that it's treated as a new cop, `VersionAdded` is set to the release version, and `VersionChanged` is reset. At least judging by [this example](https://github.com/rubocop-hq/rubocop/commit/f5b9401c4a5354b72100a8c1ffa33076942eeadc#diff-3653213e3a153018a75cc7b2536b1d0217a6d8d75188d71cf45441812f789698R3408).

Comment (bquorning) [reply config/default.yml:626]: Interesting. Another example is https://github.com/rubocop-hq/rubocop/commit/8ccdf3e1980b6349a841196f00329c12c6e6bfa1 where `VersionChanged` is updated, but `VersionAdded` is unchanged.

Comment (bquorning) [reply config/default.yml:626]: Also see config/default.yml in https://github.com/rubocop-hq/rubocop/pull/7468/files – only `VersionChanged` is updated.

Comment (bquorning) [reply config/default.yml:626]: I’ll go with changing `VersionChanged`.

**Review by jesperronn — APPROVED** [review]
Although this is my first visit to rubocop-rspec code, this looks fairly safe to change (LGTM). I also think it is a good call in the changelog to call it "potentially breaking" although it is unlikely to be clashing with other rubocop extensions.

If I understand correctly, if you have configured these cops in your own .rubocop.yml/.rubocop_todo.yml, then you MUST also rename it there.

I would reckon it would help users to call this a breaking change. Good call 👍 .

**Comment by pirj** [comment]
@jesperronn We have a comprehensive document describing all the breaking changes in the 2.0 release and the migration procedure, you may check https://github.com/rubocop-hq/rubocop-rspec/pull/1013 if you're interested.

**Pull request merged**